mvc/src/main/java/nextstep/mvc/view/JspView.java

```diff
@@ -1,30 +1,73 @@
 package nextstep.mvc.view;
 
+import jakarta.servlet.RequestDispatcher;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
+import java.util.Map;
+import nextstep.mvc.exception.view.NoSuchRequestDispatcherException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.Map;
-
 public class JspView implements View {
 
-    private static final Logger log = LoggerFactory.getLogger(JspView.class);
+    private static final Logger LOG = LoggerFactory.getLogger(JspView.class);
+    private static final String REDIRECT_PREFIX = "redirect:";
+    private static final String JSP_SUFFIX = ".jsp";
 
-    public static final String REDIRECT_PREFIX = "redirect:";
+    private final String name;
 
-    public JspView(String viewName) {
+    public JspView(String name) {
+        this.name = name;
     }
 
     @Override
-    public void render(Map<String, ?> model, HttpServletRequest request, HttpServletResponse response) throws Exception {
-        // todo
+    public void render(
+        Map<String, ?> model,
+        HttpServletRequest request,
+        HttpServletResponse response
+    ) throws Exception {
+        setAttributes(model, request);
+
+        if (name.startsWith(REDIRECT_PREFIX)) {
+            response.sendRedirect(parsePath());
+            return;
+        }
 
-        model.keySet().forEach(key -> {
-            log.debug("attribute name : {}, value : {}", key, model.get(key));
-            request.setAttribute(key, model.get(key));
+        RequestDispatcher requestDispatcher = getRequestDispatcher(request, getPath());
+        requestDispatcher.forward(request, response);
+    }
+
+    private void setAttributes(Map<String, ?> model, HttpServletRequest request) {
+        model.forEach((key, value) -> {
+            if (value != null) {
+                LOG.debug("attribute name: {}, attribute value: {}", key, model.get(key));
```

Comment:
```suggestion
                LOG.debug("attribute name: {}, attribute value: {}", key, value);
```
기본 코드였지만 이쪽도 바꿀 수 있을 것 같아요!